ABOUT.md

```diff
@@ -10,13 +10,16 @@ Try out the most recent release with Boa's live demo
 
 ## Boa Crates
 
+- [**`boa_cli`**][cli] - Boa's CLI && REPL implementation
 - [**`boa_ast`**][ast] - Boa's ECMAScript Abstract Syntax Tree.
 - [**`boa_engine`**][engine] - Boa's implementation of ECMAScript builtin objects and execution.
 - [**`boa_gc`**][gc] - Boa's garbage collector.
+- [**`boa_icu_provider`**][icu] - Boa's ICU4X data provider.
 - [**`boa_interner`**][interner] - Boa's string interner.
+- [**`boa_interop`**][interop] - Boa's interop with other languages.
```

Comment:
Hmm, you comment about the `boa_interop` crate made me reconsider its name, since it doesn't really have anything to do with interop between languages, just interop between API. Maybe we should change it to `boa_adapters`?

Reply:
Maybe be a bit more specific about Boa's Rust interop? I feel like FFI/`boa_ffi` would be pretty standard name wise for a crate that would interact with different languages.

Reply:
Current ideas:

- boa_adapters
- boa_wrappers
- boa_helpers
- boa_utils
- boa_rust_interop
- boa_bindings

Reply:
My preferred options would probably be:

- boa_rust_interop
- boa_helpers
- boa_utils (maybe `boa_js_utils`?)

I'm hesitant about using `boa_utils` if we plan to actually have a `util` directory in the future for misc crates. It may cause some confusion for newer contributors.

Reply:
`helpers` would have my vote. It's not just bindings (there are macros in there) and adapters has a specific design meaning which doesn't apply.

Also, it could just be that we don't publish it yet and just fold it into either existing packages and/or new packages. Like `boa_modules` to create modules, `boa_macros` for macro stuff, `boa_engine` for the `into_js_*_function`.

Reply:
> Also, it could just be that we don't publish it yet and just fold it into either existing packages and/or new packages. Like boa_modules to create modules, boa_macros for macro stuff, boa_engine for the into_js_*_function.

I would second this idea. We probably will split up the `boa_engine` crate further in the near future so there may be an intuitive place for all of the funtionality.

Reply:
> Also, it could just be that we don't publish it yet and just fold it into either existing packages and/or new packages. Like boa_modules to create modules, boa_macros for macro stuff, boa_engine for the into_js_*_function.

Huh, now that I think about this yeah, we could merge it into the engine crates for now and extract it in the future if people ask us to do that. This could also make it more discoverable in a way, because you won't need to pull a separate crate to have the nicer APIs to create classes, functions and methods.

Reply:
Okay I think we're in agreement about merging `boa_interop` into other crates, so I'll open an issue for it.